packages/core/src/server/devServer.ts

```diff
@@ -222,25 +223,54 @@ export async function createDevServer<
     compileMiddlewareAPI,
   });
 
+  const pwd = options.context.rootPath;
+
+  const readFileSync = (fileName: string) => {
```

Comment:
Can we use async fs API for performance?

Reply:
This API is used to hack the require function in chunk (supports reading files from rsbuild memfs), which needs to return synchronously.